### PR TITLE
Clean geomstats.errors

### PR DESCRIPTION
**Pull request by ninamiolane**
Clean geomstats.errors module:
- geomstats.error -> geomstats.errors to be consistent with geomstats.tests (plural)
- clean check_belongs method, no need for manifold's name as parameter
- add unit tests

### CODE REVIEW

File: geomstats/errors.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def check_integer(n, n_name):
                 ' got {}.'.format(n_name, n))
 
 
-def check_belongs(point, manifold, manifold_name):
+def check_belongs(point, manifold):
```
Comment (nguigs) [inline geomstats/errors.py:26]: should we allow to pass a `tol` kwarg in this method, as belong is sometimes subject to numerical precision?

Comment (ninamiolane) [reply geomstats/errors.py:26]: True. And this is where the DeepSource error "Arguments differ from overridden method" was helpful: in order to do this, we should harmonize the arguments of parents' and children's methods - even if this means that some args will be unused.

What do you think?

Comment (nguigs) [reply geomstats/errors.py:26]: In the long term yes, but this function would work anyway, if tol is only specified in classes where it needs to be, and in this method you just get **kwargs in the args and feed **kwargs to belong?

Comment (ninamiolane) [reply geomstats/errors.py:26]: Good point. I've modified it and added a test with atol.

**Pull request merged**